### PR TITLE
Adding VSCE to our publish script, and removing remnants of twitter

### PR DESCRIPTION
**Pull request by joehan**
### Description
Adding VSCE to our release script and removing remnants of broken twitter integration. Now, on every CLI release, we will cut a patch release of VSCode. If you want to make a major/minor version update, you can manually increase package.json.

### Scenarios Tested
Ran this script against a different bucket, and successfully built + uploaded the 2 VSIX files

### CODE REVIEW

File: firebase-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Change Log
```
Comment (joehan) [inline firebase-vscode/CHANGELOG.md:1]: I made this change to automate our changelog - we can put change log entries for the next release under NEXT, and the script will automatically replace it with the next version when its run.

File: scripts/publish.sh [hunk 2]
```diff
@@ -87,6 +81,10 @@ npm version $VERSION
 NEW_VERSION=$(jq -r ".version" package.json)
 echo "Made a $VERSION version."
 
+echo "Preparing VSCode extension..."
+./scripts/publish-vscode.sh patch
```
Comment (joehan) [inline scripts/publish.sh:85]: Hardcoding this to patch right now - open to switching this to match the CLI version type (ie if we release a minor version of the CLI, the accompanying VSCE release will be minor) - lmk what you prefer.

File: firebase-vscode/package.json [hunk 3]
```diff
@@ -191,12 +191,11 @@
   "scripts": {
```
Comment (hlshen) [inline firebase-vscode/package.json:191]: did my changes for this file not merge?

File: scripts/publish.sh (hunk 2 above)
Comment (hlshen) [reply scripts/publish.sh:85]: I don't think it should match. VSCode features upgrades will definitely differ.

What does the CLI do? Can we specify the update "level" through changelog?

**Review by hlshen — APPROVED** [review]

**Review by yuchenshi — APPROVED** [review]

**Review by hlshen — APPROVED** [review]

**Pull request merged**